app-crypt/mit-krb5/mit-krb5-1.18.2-r1.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Why not just `python3_7`? The original recipe does not list `python3_6` as compatible.

Reply:
We have ebuilds for 3.6 and 3.7 but only 3.6 is built at the moment.

Reply:
Oh, okay, makes sense then, thanks for elaborating. The tests show it builds and it works with `python3_6`, so we're good to go.